src/Sulu/Component/Rest/ListBuilder/ListRestHelper.php

```diff
@@ -22,6 +22,13 @@
  */
 class ListRestHelper implements ListRestHelperInterface
 {
+    const PARAMETER_FIELDS = 'fields';
+    const PARAMETER_PAGE = 'page';
+    const PARAMETER_SEARCH = 'search';
+    const PARAMETER_SEARCH_FIELDS = 'searchFields';
+    const PARAMETER_SORT_BY = 'sortBy';
+    const PARAMETER_SORT_ORDER = 'sortOrder';
```

Comment:
Would be possible, but feels a little bit dirty, doesn't it? We are bloating the constructor arguments quite a lot this way... If you would use `static` instead of `self` when accessing the constants it should be possible extend this class and introduce your own variables. Wouldn't that be better?

However, I also think that it doesn't matter too much, since this won't be used often anyway.